scripts/docgen/docgen.php

```diff
@@ -202,6 +202,26 @@ function get_type_by_string($str) { /* {{{ */
 }
 /* }}} */
 
+/** @return string|null */
+function get_type_as_string(ReflectionType $type = null) { /* {{{ */
```

Comment:
Why not just `return (string) $type` for all non-null inputs?

Reply:
Because [ReflectionType::__toString()](https://www.php.net/manual/en/reflectiontype.tostring.php) is deprecated as of PHP 7.1.0.

Reply:
Ack.